### ISSUE TITLE
Multiple authors for a single blog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#56]
```
I would like the ability to add multiple authors to a single blog. I'm running 
a metatextual fiction blog in which I post as both myself and as a fictional 
character. Each individual has a different login. I would love to be able to 
add each to ScribeFire. When I try the second signin overwrites the first.

```

-----
Original issue reported on code.google.com by `elib...@gmail.com` on 9 Jun 2010 at 4:27

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 15 Jun 2010 at 3:08
* Changed title: **Multiple authors for a single blog**
* Changed state: **Accepted**
* Added labels: **Type-Enhancement**, **Priority-Low**, **Milestone-1.1**

**Comment by GoogleCodeExporter** [comment]
```
Done in r76. This will be included in version 1.1.

Whenever you add a single blog more than once under multiple authors, the 
author username will appear in the blog list like so:

"My Blog (username @ http://example.com)"

If a blog is only listed once, the username will not be shown:

"My Blog (http://example.com)"
```

-----
Original comment by `cfinke` on 12 Jul 2010 at 3:19
* Changed state: **Fixed**
